### DISCUSSION TITLE
2022-01-07 Deploy from GitLab to APPUiO Cloud

### DISCUSSION THREAD

**Discussion by tobru** [appuio/appuio-cloud-community#34] [Announcements]
We published a guide how to connect GitLab to APPUiO Cloud: [Connect GitLab to APPUiO Cloud](https://docs.appuio.cloud/user/how-to/connect-gitlab.html). With this guide it makes it easy to deploy applications directly from GitLab to APPUiO Cloud.

**Comment by bittner** [comment]
GitLab is about to [remove certificate-based Kubernetes integration](https://about.gitlab.com/blog/2021/11/15/deprecating-the-cert-based-kubernetes-integration/#what-to-expect) starting with May this year (or later). They are encouraging customers to [install the GitLab Agent](https://about.gitlab.com/blog/2021/11/15/deprecating-the-cert-based-kubernetes-integration/#what-should-you-do-for-clusters-not-connected-to-gitlab-yet) is their clusters instead.

Will APPUiO Cloud have a (global) GitLab Agent for every customer to use out-of-the-box? Or can / should we request the installation of [an isolated instance](https://docs.gitlab.com/ee/user/clusters/agent/install/index.html#install-multiple-agents-in-your-cluster) for every project?

**Reply by tobru** [reply]
This isn't decided yet. Would you mind adding an [idea](https://github.com/appuio/appuio-cloud-community/discussions/categories/ideas) so that we can track this requirement? It feels like a valuable feature to have.

**Reply by bittner** [reply]
Done. https://github.com/appuio/appuio-cloud-community/discussions/41
